assets/jss/app.js

```diff
@@ -0,0 +1,104 @@
+/*const contactNav = document.getElementById("contactNav");
+const aboutNav = document.getElementById("aboutNav");
+const aboutMe = document.getElementsByClassName("p-text")[0];
+const contactForm = document.getElementsByClassName('contact-form')[0];
+
+console.log("works");
+
+aboutNav.addEventListener("click", function () {
+    console.log("works1");
+    contactForm.style.display = "none"
+    aboutMe.style.display = "block"
+});
+
+contactNav.addEventListener("click", function () {
+    console.log("works1");
+    aboutMe.style.display = "none"
+    contactForm.style.display = "block"
+});
+*/
+
+//object with all elments of the page
+//elmebtsbyclassname returns a node list so we need to grab inital index similar to array
+const elements = {
+  home: document.getElementsByClassName("home")[0],
+  about: document.getElementsByClassName("p-text")[0],
+  contact: document.getElementsByClassName("contact-form")[0],
+  swift: document.getElementsByClassName("circle-1")[0],
+  vueJS: document.getElementsByClassName("circle-2")[0],
+  netlify: document.getElementsByClassName("circle-3")[0]
+};
+
+//navbar elements
+const nav = {
+  home: document.getElementById("homeNav"),
+  about: document.getElementById("aboutNav"),
+  contact: document.getElementById("contactNav")
+};
+
+//functions for showing and hiding elements
+const showHome = homeElement => {
+  homeElement.style.display = "flex";

```

Comment:
Don't set CSS manually in JS. Instead have 2 CSS classes that you switch. Let me know if you need clarification.